rbac/internal/migrations/replicate_workspaces.py

```
@@ -262,5 +278,23 @@ def replicate_deleted_workspaces(since: datetime.datetime, replicator: Optional[
+
     for batch in itertools.batched(delete_logs.iterator(), 500):
-        _replicate_deleted_batch(replicator, [_DeletedWorkspaceEntry(l.tenant_id, l.resource_uuid) for l in batch])
+        _replicate_deleted_batch(
+            replicator,
+            [
+                _DeletedWorkspaceEntry(
+                    tenant_id=l.tenant_id,
+                    workspace_id=l.resource_uuid,
+                    workspace_name=_extract_deleted_workspace_name(l),
+                )
+                for l in batch
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)